openid4vc-high-assurance-interoperability-profile-sd-jwt-vc-1_0.md

```diff
@@ -79,6 +84,9 @@ The following items are out of scope for the current version of this document, b
 
 * Trust Management, i.e. authorization of an issuer to issue certain types of credentials, authorization of the Wallet to be issued certain types of credentials, authorization of  the Verifier to receive certain types of credentials.
 * Protocol for presentation of Verifiable Credentials for offline use-cases, e.g. over BLE.
+* Profile of OID4VCI to issue ISO mdocs is defined in [@!ISO.23220-3].
+* Profile of OID4VP without using W3C Digital Credentials API to present ISO mdocs is 
+defined in [@!ISO.18013-7]. For more details, also see Annex B.3 in [@!OIDF.OID4VP].
```

Comment:
I was under the impression this draft would replace the -7 OID4VP mdoc profile. If not, how is the synchronization between this draft and changes done at ISO on -7 achieved?

Reply:
my understanding was that 
- since 18013-7 has been published, it will continue to define how to do mdocs over vanilla OID4VP
- this profile defines mdocs over OID4VP with browser API